packages/cli/src/lib/docker-proxy-client.ts

```diff
@@ -12,7 +12,6 @@ const DOCKER_PROXY_PORT = 3000
 const baseDockerProxyService: ComposeService = {
   build: {
     context: DOCKER_PROXY_DIR,
```

Comment:
We might need to add `target: 'production'`

Reply:
the default is the last builder